index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
```

Comment:
`SUGGESTION`

Consider using the `lang="en"` attribute for the `html` start tag, this is good for internationalization. You should check this [link for more info](https://www.w3.org/International/questions/qa-html-language-declarations#basics).

Reply:
Thank you for the info, I didn't know that. Suggestion done.